perfmetrics/scripts/continuous_test/ml_tests/tf/setup_scripts/setup_container.sh

```diff
@@ -0,0 +1,157 @@
+) -> Tuple[tf.keras.Model, Mapping[str, Any]]:
+  \"\"\"Runs train/eval configured by the experiment params.
+
+  Args:
+    distribution_strategy: A distribution distribution_strategy.
+    task: A Task instance.
+    mode: A 'str', specifying the mode. Can be 'train', 'eval', 'train_and_eval'
+      or 'continuous_eval'.
+    params: ExperimentConfig instance.
+    model_dir: A 'str', a path to store model checkpoints and summaries.
+    run_post_eval: Whether to run post eval once after training, metrics logs
+      are returned.
+    save_summary: Whether to save train and validation summary.
+    train_actions: Optional list of Orbit train actions.
+    eval_actions: Optional list of Orbit eval actions.
+    trainer: the base_trainer.Trainer instance. It should be created within the
+      strategy.scope().
+    controller_cls: The controller class to manage the train and eval process.
+      Must be a orbit.Controller subclass.
+
+  Returns:
+    A 2-tuple of (model, eval_logs).
+      model: \`tf.keras.Model\` instance.
+      eval_logs: returns eval metrics logs when run_post_eval is set to True,
+        otherwise, returns {}.
+  \"\"\"
+  runner = OrbitExperimentRunner(
+      distribution_strategy=distribution_strategy,
+      task=task,
+      mode=mode,
+      params=params,
+      model_dir=model_dir,
+      run_post_eval=run_post_eval,
+      save_summary=save_summary,
+      train_actions=train_actions,
+      eval_actions=eval_actions,
+      trainer=trainer,
+      controller_cls=controller_cls,
+  )
+  return runner.run(epochs=epochs, clear_kernel_cache=clear_kernel_cache)
+" > bypassed_code.py
+
+train_lib_file="/root/.local/lib/python3.7/site-packages/official/core/train_lib.py"
+x=$(grep -n "def run_experiment(" $train_lib_file | cut -f1 -d ':')
+y=$(grep -n "return runner.run()" $train_lib_file | cut -f1 -d ':')
+lines="$x,$y"
+sed -i "$lines"'d' $train_lib_file
+x=$((x-1))
+sed -i "$x"'r bypassed_code.py' $train_lib_file
+
+echo "  def run(self, epochs=1, clear_kernel_cache=False) -> Tuple[tf.keras.Model, Mapping[str, Any]]:" > bypassed_code.py
+x=$(grep -n "def run(self) -> Tuple\[tf.keras.Model, Mapping\[str, Any\]\]:" $train_lib_file | cut -f1 -d ':')
+lines="$x,$x"
+sed -i "$lines"'d' $train_lib_file
+x=$((x-1))
+sed -i "$x"'r bypassed_code.py' $train_lib_file
+
+echo "
+      if mode == 'train' or mode == 'train_and_post_eval':
+        self.controller.train(steps=params.trainer.train_steps, clear_kernel_cache=clear_kernel_cache, epochs=epochs)" > bypassed_code.py
+x=$(grep -n "if mode == 'train' or mode == 'train_and_post_eval':" $train_lib_file | cut -f1 -d ':')
+y=$(grep -n "self.controller.train(steps=params.trainer.train_steps)" $train_lib_file | cut -f1 -d ':')
+lines="$x,$y"
+sed -i "$lines"'d' $train_lib_file
+x=$((x-1))
+sed -i "$x"'r bypassed_code.py' $train_lib_file
+
+# Start training the model
+#nohup python3 -u resnet.py > /home/output/myprogram.out 2> /home/output/myprogram.err &
+
+# TODO cron job
```

Comment:
adding comment so that it's not skipped.

Reply:
Cron can be added to VM as well because the directory containing logs is shared. Please adopt whichever way is easier.